src/main/java/io/github/thebusybiscuit/slimefun4/utils/ChatUtils.java

```diff
@@ -80,4 +80,21 @@ public static void awaitInput(@Nonnull Player p, @Nonnull Consumer<String> callb
         return builder.toString();
     }
 
+    /**
+     * This method adds an s to a string if the supplied integer is not 1.
+     *
+     * @param string
+     *      The string to potentially pluralize
+     * @param count
+     *      The amount of things
+     * @return
+     *      <code>string</code> if <code>count</code> is 1 else <code>string + "s"</code>
+     */
+    public static @Nonnull String checkPlurality(@Nonnull String string, int count) {
+        if (count == 1) {
+            return string;
+        }
+        return string + "s";
```

Comment:
```suggestion
        return count == 1 ? string : string + "s";
```
Ik it's ironic because of the PR name

Reply:
can I get a second opinion on this? Sure, it's a return statement, but style guide still says minimize amount of ternary operators.

Reply:
I prefer the one without ternary tbh. Im not a fan of them at all tbh.

Reply:
agreed